README.md

```diff
@@ -42,8 +42,8 @@ ArcticDB is designed from the outset to be resilient; there is no single point o
 
 |                       | PyPI (Python 3.6 - 3.11) | conda-forge (Python 3.8 - 3.11) |
 | --------------------- | - | - |
-| Linux                 | ✔️ | ✔️ |
-| Windows               | ✔️ | ➖ |
+| Linux (Intel/AMD)     | ✔️ | ✔️ |
+| Windows (Intel/AMD)   | ✔️ | ➖ |
```

Comment:
It is worth mentioning it, yes.

FYI, [conda-forge build status](https://github.com/conda-forge/arcticdb-feedstock/?tab=readme-ov-file#current-build-status) might also be useful if users want to fully understand which combinations are available and request the support of other architectures (e.g. ARM's).

Reply:
I've added a link to it.